go-controller/pkg/ovn/policy.go

```diff
@@ -222,33 +223,67 @@ func deleteACLPortGroup(portGroupName, direction, priority, match, action string
 	return nil
 }
 
-func addToPortGroup(ovnNBClient goovn.Client, portGroupName string, portInfo *lpInfo) error {
-	cmd, err := ovnNBClient.PortGroupAddPort(portGroupName, portInfo.uuid)
-	if err == nil {
-		if err = ovnNBClient.Execute(cmd); err != nil {
-			return fmt.Errorf("execute error adding port %s to port group %s (%v)", portInfo.name, portGroupName, err)
+func addToPortGroup(ovnNBClient goovn.Client, portGroupName string, ports ...*lpInfo) error {
+	cmds := make([]*goovn.OvnCommand, 0, len(ports))
+	for _, portInfo := range ports {
+		cmd, err := ovnNBClient.PortGroupAddPort(portGroupName, portInfo.uuid)
+		if err != nil {
+			return fmt.Errorf("error preparing adding port %v to port group %s (%v)", portInfo.name, portGroupName, err)
 		}
-	} else if err != goovn.ErrorExist {
```

Comment:
why did you get rid of checking if exists? @squeed

Reply:
@trozet because I changed go-ovn to stop artificially generating ErrorExist.